acceptance/tests/catalog/replace_catalog_debugging.rb

```diff
@@ -92,5 +92,33 @@
 
   end
 
+  step "relationship on resource alias" do
+
+    manifest = <<MANIFEST
+
+    package { foo:
+      ensure => installed,
+      alias=>bar,
+    }
+    service{ baz:
+      ensure => running,
+      require => Package[bar]
+    }
+
+MANIFEST
+
+    run_agents_with_new_site_pp(master, manifest, {}, [0,2,4])
```

Comment:
@wkalt not ... really ... I guess why I was suggesting the noop approach, would make it never fail - since it's a noop. ie. the problem might go away. Also it seems more correct anyway?